src/common/i18n/translations/zh.json

```diff
@@ -0,0 +1,363 @@
+{
+  "loginScanCard": {
+    "loginWithQR": "请使用您专属的QR码登录",
+    "scanToLogin": "扫描QR码",
+    "needHelp": "需要协助？",
+    "askQuestion": "问问题"
+  },
+  "loginMobileNumberCard": {
+    "enterMobileNumber": "请输入您的手机号码，获取一次性密码",
+    "mobileNumber": "手机号码",
+    "sendOtp": "发送一次性密码"
+  },
+  "loginOTPCard": {
+    "sendingOtp": "一次性密码发送中...",
+    "Otp": "一次性密码",
+    "resendIn": "在%{ss}秒内重新发送",
+    "submit": "提交",
+    "resend": "重新发送"
+  },
+  "collectCustomerDetailsScreen": {
+    "checkEligibleItems": "查看可领取的物品数量",
+    "scanIdentification": "扫描证件",
+    "or": "或",
+    "enterIdNumber": "输入证件号码",
+    "check": "查看"
+  },
+  "idScanner": {
+    "enterIdManually": "输入证件号码",
+    "enterManually": "自己输入",
+    "enterVoucherId": "输入优惠券编号",
+    "scanToCheck": "扫描查看有效"
```

Comment:
Was this in the translation docs? Don't rmb seeing it leh. Doesn't sound accurate 😆 Which screen is this in?

Reply:
HAHA scan to check was needed in the voucher scanner screen and I was desperate😂